.gitpod.yml

```diff
@@ -9,10 +11,11 @@ ports:
 tasks:
   - init: |
       npm install
-      npx prisma db push
```

Comment:
No longer needed, we're using the `predev` script to keep the DB in sync with the schema whenever we (re)start the dev server.